docs/guides/migrating-to-v2.mdx

```diff
@@ -57,51 +57,37 @@ const Component = () => {
 }
 ```
 
+```diff
```

Comment:
You are overwriting the change in the main branch.

Reply:
Woops - not sure what happened there :\